Rakefile

```diff
@@ -87,7 +87,17 @@ namespace :dependencies do
     task :apply do
       next unless Dir.exist?(File.join(Dir.home, '.mobile-secrets/.git')) || ENV.key?('CONFIGURE_ENCRYPTION_KEY')
 
-      sh('FASTLANE_SKIP_UPDATE_CHECK=1 FASTLANE_ENV_PRINTER=1 bundle exec fastlane run configure_apply force:true')
+      command_compontents = %w[
+        FASTLANE_SKIP_UPDATE_CHECK=1
+        FASTLANE_HIDE_CHANGELOG=1
+        FASTLANE_HIDE_PLUGINS_TABLE=1
+        FASTLANE_ENV_PRINTER=1
+        FASTLANE_SKIP_ACTION_SUMMARY=1
+        FASTLANE_HIDE_TIMESTAMP=1
+        bundle exec fastlane run configure_apply force:true
+      ]
+
+      sh(command_compontents.join(' '))
```

Comment:
I couldn't get the [suggested](https://github.com/wordpress-mobile/WordPress-iOS/pull/19088/files#r927130041) `shelljoin` to work, but maybe this is a better alternative, because it doesn't require an additional method call

```suggestion
     # The string is indented all the way to the left to avoid padding when printed in the terminal 
      command = %(
FASTLANE_SKIP_UPDATE_CHECK=1 \
FASTLANE_HIDE_CHANGELOG=1 \
FASTLANE_HIDE_PLUGINS_TABLE=1 \
FASTLANE_ENV_PRINTER=1 \
FASTLANE_SKIP_ACTION_SUMMARY=1 \
FASTLANE_HIDE_TIMESTAMP=1 \
bundle exec fastlane run configure_apply force:true
      )

      sh(command)
```

Result:

![image](https://user-images.githubusercontent.com/1218433/180366021-557c579b-eb93-4704-8fcb-7f073b54ac80.png)

Reply:
Oh sorry, I missed the env vars which were treated as part of the shell script by the `shelljoin` function.

Your suggested changes here look good to me. The indentation looks a bit strange though, you can potentially rewrite it to something like this:
```ruby
sh(
  {
    'FASTLANE_ENV' => '1',
    ...
  },
  %(bundle exec ...)
)
```

I don't really have a strong opinion on this. They all look quite similar 😸

Reply:
No worries. I like your suggestion with an hash of env vars, but I can't get it to work 😞 

I'm going to use the version from my [suggestion above](https://github.com/wordpress-mobile/WordPress-iOS/pull/19088#discussion_r927290096) which also has a note about the awkward indentation. I'm keen to iterate on this, but also don't want it to get stale.

As an aside, I question the point of having `Rakefile` and `Fastfile` as they both serve a similar purpose, namely task automation.

In the long run, we'd like to remove the need for feature devs to deal with Ruby. Under that assumption, we might want to limit the time we spend on `Rakefile` tweaks.

The Pocket Casts iOS folks are using a `Makefile`, which I think it's pretty cool and more lightweight.